frontend/src/pages/search/index.tsx

```diff
@@ -184,21 +184,18 @@ const Search = () => {
 
     // keep panel open if clicking a different document
     if (document?.document_id != id) {
-      // setShowSlideout(true);
       resetSlideOut(true);
     } else {
-      // setShowSlideout(!showSlideout);
       resetSlideOut();
     }
     updateDocument.mutate(id);
-
-    // setShowPDF(false);
   };
 
   const getCurrentSortChoice = () => {
     const field = searchCriteria.sort_field;
     const order = searchCriteria.sort_order;
     if (field === null && order === "desc") {
+      if (browsing) return "date:desc";
       return "relevance";
```

Comment:
Just one question, do we also hide the option "Relevance" when no search term is provided?

Reply:
Yes good question - if you look in the `filters/Sort.tsx` component you can see I'm selecting the sort options based on whether we are in 'browsing' mode or not.

Reply:
cheers!